test/fixtures/ledger_transaction_fixtures.js

```diff
@@ -91,7 +105,7 @@ const buildTransactionDetails = (opts = {}) => {
       status: opts.refund_summary_status || 'unavailable',
       user_external_id: opts.user_external_id || null,
       amount_available: opts.refund_summary_available || 20000,
-      amount_submitted: opts.refund_summary_submitted || 0
+      amount_refunded: opts.refund_summary_submitted || 0
```

Comment:
Maybe the option should be renamed as well (unless it's not ledger test specific).